pkg/cmd/cmd.go

```diff
@@ -83,12 +94,41 @@ func newFullCoverageCommand() *cobra.Command {
 					float64(info.TotalCoveredLines)/float64(info.TotalLines)*100,
 				)
 			}
+
+			if kustoOption.UseKusto {
+				dbClient, err := dbclient.NewKustoClient(kustoOption)
+				if err != nil {
+					return fmt.Errorf("new kusto client: %w", err)
+				}
+
+				now := time.Now().UTC()
+				for _, info := range all {
+					err = dbClient.Store(context.Background(), &dbclient.Data{
+						PreciseTimestamp: now,
+						LinesCovered:     info.TotalCoveredLines,
+						LinesValid:       info.TotalLines,
+						ModulePath:       moduleHostPath,
+						FilePath:         info.Path,
+						Coverage:         float64(info.TotalCoveredLines) / float64(info.TotalLines),
+					})
+					if err != nil {
+						return fmt.Errorf("store data: %w", err)
+					}
+				}
+
+			}
+
 			return nil
 		},
 	}
 
 	cmd.Flags().StringVar(&coverProfile, "cover-profile", "", `coverage profile produced by 'go test'`)
 	cmd.Flags().StringVar(&moduleHostPath, "host-path", "", "host path for the go project")
+	cmd.Flags().BoolVar(&kustoOption.UseKusto, "use-kusto", false, "whether enable kusto to collect coverage data")
+	cmd.Flags().StringVar(&kustoOption.Endpoint, "endpoint", "", "kusto endpoint")
+	cmd.Flags().StringVar(&kustoOption.Database, "database", "", "kusto database")
+	cmd.Flags().StringVar(&kustoOption.Event, "event", "", "kusto event")
+	cmd.Flags().StringSliceVar(&kustoOption.Overrides, "overrides", []string{}, "kusto column overrides, format: {column}:{datatype}:{value}")
```

Comment:
what's this for?

Reply:
Extra data sent to kusto. Data in project gocover only contains domain unrelated fundamental fields. This `overrides` can pass that information, for example, pullrequestid, buildid from azuredevops.

Reply:
can we change this flag name to "custom-columns" ? the name "overrides" seems like to replace the exist columns.

Reply:
update